addons/mail/static/src/core/common/chat_hub.scss

```diff
@@ -5,6 +5,10 @@
     margin-bottom: $o-mail-ChatHub-bubblesMargin;
     margin-left: $o-mail-ChatHub-bubblesMargin;
     z-index:  $o-mail-NavigableList-zIndex - 1;
+
+    &.o-mail-ChatHub-lift {
```

Comment:
can simply be named `o-liftUp` (we see it's a modifier classname, less intimidating to read